opts/fixtures/envdir/junk/.FOO

```diff
@@ -0,0 +1 @@
+bar
```

Comment:
change this to something other than 'bar' otherwise we won't know if it picked it up since it will overlap with the "FOO" file's value.